lib/Serialization/Deserialization.cpp

```diff
@@ -2039,6 +2039,24 @@ ModuleFile::resolveCrossReference(ModuleID MID, uint32_t pathLen) {
     }
     filterValues(filterTy, nullptr, nullptr, isType, inProtocolExt,
                  importedFromClang, isStatic, std::nullopt, values);
+    if (values.empty() && importedFromClang && name.isOperator() && filterTy) {
```

Comment:
It might be helpful for readers like me if this comment has examples of a "Clang-importer instantiated/synthesized conformance operator" (I think it it refers to `==`, `+=` etc.?)